src/Microsoft.ML.Transforms/Text/LineParser.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Globalization;
+using System.Linq;
+
+namespace Microsoft.ML.Transforms.Text
+{
+    public static class LineParser
+    {
+        private static readonly char[] _delimiters = { ' ', '\t' };
+
+#if NETSTANDARD2_0
```

Comment:
>NETSTANDARD2_0 [](start = 4, length = 14)

Such broad divergences in implementation make me a little nervous, since unless I'm very much mistaken we don't actually have multiple configurations of whether we have one version of .NET standard or another. Is this separation absolutely urgent? If it has to do with the presence or absence of methods of span of char, could ReadOnlyMemoryUtilities be made to serve?